remoteit/desktop
# [DESK-650] Reinstall cli

### Description


Fix the re-install CLI button

### Test plan



1. Sign in
2. Go to settings page
3. Click re-install command line tools
4. You should see the tools get re-installed

### Ticket



<https://remoteit.atlassian.net/browse/[DESK-650]>


[DESK-650]: https://remoteit.atlassian.net/browse/DESK-650
